tests/mammal.test.ts

```diff
@@ -0,0 +1,28 @@
+import Lion from "../src/animals/Lion";
```

Comment:
if you have a look at the main branch, there is a reptiles folder created for all the reptiles tests. you should follow the structure and create a mammals folder with the animals in it.